lib/Sema/CSBindings.cpp

```diff
@@ -1620,19 +1620,29 @@ void BindingSet::dump(llvm::raw_ostream &out, unsigned indent) const {
   PO.PrintTypesForDebugging = true;
 
   out.indent(indent);
+  std::vector<StringRef> attributes;
```

Comment:
Note - I think it's going to be less error-prone to use `std::string` here for the element type because it manages the storage.